fix: Methods must be under instance methods (#38858)
mdn/content: files/en-us/web/api/svgelement/index.md

> _This interface has no methods, but inherits methods from {{DOMxRef("Element")}}._

Reviewer: ```suggestion
_This interface also inherits methods from {{DOMxRef("Element")}}._
```

> - {{DOMxRef("SVGElement.blur")}}
>   - : Removes keyboard focus from the currently focused element.
> - {{DOMxRef("SVGElement.focus")}}
>   - : Makes the element the current keyboard focus.

Reviewer: ```suggestion
- {{DOMxRef("SVGElement.blur()")}}
  - : Removes keyboard focus from the currently focused element.
- {{DOMxRef("SVGElement.focus()")}}
  - : Makes the element the current keyboard focus.
```
